duckduckgo/iOS
# show nicer message when no privacy practices are found

Reviewer: Caine or Mia
Asana: https://app.asana.com/0/414235014887631/487711692211748
CC:

**Description**:

Show a nicer message when there are unknown privacy practices.

**Steps to test this PR**:
1. Visit a site that has unknown privacy practices, e.g. falkirkrpg.org.uk and observe new friendly message
1. Visit a site with privacy practices evaluated (e.g. google) and observe list of privacy practices
